docs/azure/sdk/authentication/additional-methods.md

```diff
@@ -36,6 +36,64 @@ foreach (var blobItem in client.GetBlobContainers())
 
 For more exact control, such as setting redirect URIs, you can supply specific arguments to `InteractiveBrowserCredential` such as `redirect_uri`.
 
+### Authenticate the default system account via WAM
+
+Web Account Manager (WAM) is a system authentication broker service that allows apps to seamlessly request OAuth tokens from identity providers, such as Microsoft Entra ID. WAM enables identity providers to natively plug into the OS and provide the service to other apps to streamline the login process. WAM offers the following benefits:
```

Comment:
It's important to call out right away that this feature is for Windows only. We don't yet have a solution for macOS and Linux.
```suggestion
Web Account Manager (WAM) is a system authentication broker service for Windows that allows apps to seamlessly request OAuth tokens from identity providers, such as Microsoft Entra ID. WAM enables identity providers to natively plug into the OS and provide the service to other apps to streamline the login process. WAM offers the following benefits:
```